internal/bundlereader/resources.go

```diff
@@ -258,12 +258,22 @@ func loadDirectories(ctx context.Context, opts loadOpts, directories ...director
 
 	eg, ctx := errgroup.WithContext(ctx)
 
+	alreadyLoaded := make(map[string]directory)
```

Comment:
The values in this map are not actually used, consider using an empty `struct{}` or the `sets` package from Kubernetes utils.

Reply:
changed to use en empty struct.